xc/device.ml

```diff
@@ -1244,8 +1244,10 @@ let clean_shutdown (task: Xenops_task.t) ~xs (x: device) =
 			try (Xenctrl.domain_getinfo xc x.frontend.domid).Xenctrl.hvm_guest
 			with _ -> false
 			in
-		try release ~xc ~xs ~hvm devs x.frontend.domid x.frontend.devid
-		with _ -> ());
+		if hvm then 
```

Comment:
@robhoes I added that because PCI.unplug for PV guests is unsupported as in https://github.com/xapi-project/xenopsd/blob/master/xc/xenops_server_xen.ml#L1582-L1584